src/instances.ts

```diff
@@ -205,11 +205,11 @@ function successfulTypeScriptInstance(
   let normalizedFilePath: string;
   try {
     const filesToLoad = loaderOptions.onlyCompileBundledFiles
-      ? configParseResult.fileNames.filter(fileName =>
+      ? configParseResult.fileNames.filter((fileName: string) =>
           dtsDtsxRegex.test(fileName)
         )
       : configParseResult.fileNames;
-    filesToLoad.forEach(filePath => {
+    filesToLoad.forEach((filePath: string) => {
```

Comment:
is the type annotation necessary? Presumably the type can be inferred?

Reply:
Fixed